Classes/model/CoreData/Alert.h

```diff
@@ -31,6 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, retain, nullable) NSDate * startTime;
 @property (nonatomic, retain, nullable) NSDate * endTime;
 @property (nonatomic, assign) BOOL toDelete;
+@property (nonatomic, retain, nullable) NSDictionary *action;
```

Comment:
What kind of dictionary is this? `<String, id>`?

Reply:
Done